website/docs/index.html.markdown

```diff
@@ -221,7 +221,15 @@ The following arguments are used to configure the VMware NSX-T Provider:
   partially successful realization as valid state and not fail apply.
 * `vmc_token` - (Optional) Long-lived API token for authenticating with VMware
   Cloud Services APIs. This token will be used to short-lived token that is
-  needed to communicate with NSX Manager in VMC environment.
+  needed to communicate with NSX Manager in VMC environment. In conflict with 
```

Comment:
Just cosmetic comment - I'm not sure `in conflict` is a term the user will always understand. Perhaps `can not be specified together with XXX`?